### PR TITLE
LSR-958 use connectors instead of services directly

### PR DESCRIPTION
**Pull request by nhpatt**

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/util/ServiceVersionFactory62.java [hunk 1]
```diff
@@ -77,4 +77,14 @@ public JournalContentConnector getJournalContentConnector(Session session) {
 	public ScreensJournalContentConnector getScreensJournalContentConnector(Session session) {
 		return new ScreensJournalContentConnector62(session);
 	}
+
+	@Override
```
Comment (dgarciasarai) [inline android/library/src/main/java/com/liferay/mobile/screens/util/ServiceVersionFactory62.java]: It's necessary the annotation?

File: android/library/src/main/java/com/liferay/mobile/screens/util/ServiceVersionFactory70.java [hunk 2]
```diff
@@ -76,4 +76,14 @@ public JournalContentConnector getJournalContentConnector(Session session) {
 	public ScreensJournalContentConnector getScreensJournalContentConnector(Session session) {
 		return new ScreensJournalContentConnector70(session);
 	}
+
+	@Override
```
Comment (dgarciasarai) [inline android/library/src/main/java/com/liferay/mobile/screens/util/ServiceVersionFactory70.java]: The same here

File: android/library/src/main/java/com/liferay/mobile/screens/util/ServiceVersionFactory62.java (hunk 1 above)
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/util/ServiceVersionFactory62.java]: No, but it's recommended: http://stackoverflow.com/questions/212614/should-we-override-an-interfaces-method-implementation

Comment (dgarciasarai) [inline android/library/src/main/java/com/liferay/mobile/screens/util/ServiceVersionFactory62.java]: 👍

**Pull request merged**